src/Form/AgendaFilterType.php

```diff
@@ -46,13 +46,31 @@ public function configureOptions(OptionsResolver $resolver)
             'csrf_protection' => false,
             'validation_groups' => ['filtering'],
             'municipality' => null,
+            'isBoardMember' => null,
         ]);
     }
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         /** @var Municipality $municipality */
         $municipality = $options['municipality'];
+        $isBoardMember = $options['isBoardMember'];
+
+        // Modify status filter choices if board member
+        if ($isBoardMember) {
+            $statusChoices = [
+                $this->translator->trans('Finished', [], 'agenda') => AgendaStatus::FINISHED,
+                $this->translator->trans('Open', [], 'agenda') => AgendaStatus::NOT_FINISHED,
+            ];
+        } else {
+            $statusChoices = [
+                $this->translator->trans('Open', [], 'agenda') => AgendaStatus::OPEN,
+                $this->translator->trans('Full', [], 'agenda') => AgendaStatus::FULL,
+                $this->translator->trans('Ready', [], 'agenda') => AgendaStatus::READY,
+                $this->translator->trans('Finished', [], 'agenda') => AgendaStatus::FINISHED,
+                $this->translator->trans('Not-closed', [], 'agenda') => AgendaStatus::NOT_FINISHED,
+            ];
+        }
```

Comment:
Should be written as

```php
$statusChoices = $isBoardMember
  ? […]
  : […]
```

Reply:
Fixed in 20abc6d0e1bf0336ec75f36b6613c018bc759b2c.